ui/app/pages/swaps/view-quote/view-quote.js

```diff
@@ -494,6 +496,15 @@ export default function ViewQuote () {
             <i className="fa fa-arrow-right" />
           </div>
         </div>
+        <div className="view-quote__metamask-rate">
+          <div className="view-quote__metamask-rate-text">{ t('swapQuoteIncludesRate', [metaMaskFee]) }</div>
+          <div className="view-quote__metamask-rate-info-icon">
```

Comment:
req: collapse this extra div into the div that will be rendered around the tooltip by using `wrapperClassName` from `InfoTooltip`